src/benchmarks/micro/libraries/System.Net.Http/Perf.WinHttpHandler.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Net.Security;
+using System.Net.Sockets;
+using System.Security.Authentication;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.Versioning;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Net.Http.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.NoWASM)]
+    [SupportedOSPlatform("windows")]
```

Comment:
```suggestion
    [SupportedOSPlatform("windows")]
    [OperatingSystemsFilter(allowed: true, platforms: OS.Windows)]
```

The net462 builds are failing because this attribute and the `WriteAsync` overload used below don't exist there.

You could disable this benchmark on Framework like we do for SocketsHttpHandler:
https://github.com/dotnet/performance/blob/4983ccd9730d24bcffb036b26f0f4f05ea94c636/src/benchmarks/micro/MicroBenchmarks.csproj#L123-L125